pandas/tests/io/formats/test_format.py

```diff
@@ -368,6 +368,23 @@ def test_repr_min_rows(self):
             assert ".." not in repr(df)
             assert ".." not in df._repr_html_()
 
+    @pytest.mark.parametrize(
+        "data, format_option, expected_values",
+        [
+            ({"A": [12345.6789]}, "{:12.3f}", "12345.679"),
```

Comment:
Nit: Could you make `data` here the scalar values e.g. `12345.6789` and then in the test body construct `df = DataFrame({"A": [data]})`?